src/pages/ScholarshipListPage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import scholarships from '../test/scholarshipsTest';
+import ScholarshipList from '../components/ScholarshipList';
+
+function createScholarshipList(scholarship){
+    return (
+        <ScholarshipList 
+        id={scholarship.id}
+        name={scholarship.name}
+        amount={scholarship.amount}
+        deadline={scholarship.deadline}
+        website={scholarship.website}
+        />
+    );
+}
+
+function ScholarshipListView(){
```

Comment:
*ScholarshipsListView

here and elsewhere where a list of scholarships is referred to